### ISSUE TITLE
Final Peer Review - jl2872

### ISSUE THREAD

**Issue by LiJenny** [yh854/ORIE4741_project#15]
The report investigates data collected from LOL game plays, and aims to discover insights to factors that might lead to higher probability of winning a game. The dataset used is from Kaggle, with around 1,000,000 rows and 76 features.

I like the fact that the report explored a diverse range of models, including many that are not covered in class. The portion of the report that went into discussion of the production in real-world is very interesting, and provided a lot of insight as to the usability of the model. Finally, the overall prediction power of the model is very strong in most metrics, beside the  champion selection predicting victory. 

One potential improvement might be to experiment with different regularizers. With the logistic loss function, the use of regularizers might lead to even better performance. Also, perhaps trying different loss functions would be more ideal compared to using a decision tree/random forest. Since the number of rows of data outweigh the number of features, using a decision tree might lead to a high possibility of overfitting. Although, as indicated in the report, experimenting with other loss functions might lead to higher computation time, it would be nice to explore them at first, and then select the model with the best balance between computation time and performance. Another possible track to explore is to include more visuals for the performance of the model. For example, plotting the predicted value vs the actual value might lead to more insight to where the majority of the error is coming from (e.g. from outliers, from data in a specific range, etc.). The report is also lacking discussion of whether the model would be a weapon of math destruction.
